core/src/Cuda/Kokkos_Cuda_abort.hpp

```diff
@@ -65,7 +65,11 @@ namespace Kokkos {
 namespace Impl {
 
 #if !defined(__APPLE__)
+#if defined(KOKKOS_ENABLE_DEBUG_BOUNDS_CHECK)
```

Comment:
Please comment above this line that this was causing issues with the random number generator unit tests on pre-Volta GPU architectures.